### DISCUSSION TITLE
[Feature] show people age in Android app

### DISCUSSION THREAD

**Discussion by tuxmartin** [immich-app/immich#13143] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When I open a photo on the web, I see people's age at the time they were taken.

In the Android application, I see only recognized people in the photos, but their age is not displayed. Can you please add the age?

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by VenomousViper27** [comment]
I agree with this request for iOS as well. On iOS app the age is not displayed in the information panel.

**Comment by rsmanning** [comment]
Yes please! If it's not too much trouble, tag face and add description on mobile too.

**Reply by YarosMallorca** [reply]
Photo descriptions are already available on mobile.
Tags and adding faces are not yet.

**Comment by YarosMallorca** [comment]
Implemented in #16728 and fixed in #16833
Will be in the next release!
